pkg/cnb/project_descriptor_test.go

```diff
@@ -42,6 +43,19 @@ func testProcessProjectDescriptor(t *testing.T, when spec.G, it spec.S) {
 		os.RemoveAll(platformDir)
 	})
 
+	when("unsupported project descriptor version", func() {
+		it.Before(func() {
+			ioutil.WriteFile(projectToml, []byte(`
+[_]
+schema-version = "0.3"
```

Comment:
maybe make this 0.99 or something because 0.3 will be supported next and this will need to be changed